score-web/src/app/authentication/auth.service.ts

```diff
@@ -301,3 +301,18 @@ export class CanActivateUser implements CanActivate {
   }
 
 }
+
+@Injectable()
+export class CanActivateTenantInstance implements CanActivate {
+
+  constructor(private authService: AuthService) {
+  }
+
+  canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot):
+    Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
+
+    const userToken = this.authService.getUserToken();
+    return userToken.isTenantInstance;
+  }
+
+}
```

Comment:
Can you add a newline at the end of the file? A couple of other files need that fixed as well (GitHub marks them).